app/javascript/redux/reducers/greetingReducer.js

```diff
@@ -0,0 +1,18 @@
+const initialState = {
+    greeting: '',
+  };
+  
+  const greetingReducer = (state = initialState, action) => {
+    switch (action.type) {
+      case 'SET_GREETING':
+        return {
+          ...state,
+          greeting: action.payload,
```

Comment:
* Kindly Create a reducer that will connect to your API endpoint to get the random greeting as currently your react app isn't connected to your API's endpoint [Kindly refer back to this lesson on how to properly setup your project](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/lessons/rails_and_webpack.md)  [also check requirement](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/rails_react_webpack.md#:~:text=Create%20a%20store%2C%20an%20action%2C%20and%20a%20reducer%20that%20will%20connect%20to%20your%20API%20endpoint%20to%20get%20the%20random%20greeting.)